app/src/main/kotlin/com/wire/android/ui/home/conversations/MessageComposerViewModel.kt

```diff
@@ -326,8 +326,9 @@ class MessageComposerViewModel @Inject constructor(
 
     private fun setFileSharingStatus() {
         viewModelScope.launch {
-            if (isFileSharingEnabled().isFileSharingEnabled != null) {
-                conversationViewState = conversationViewState.copy(isFileSharingEnabled = isFileSharingEnabled().isFileSharingEnabled!!)
+            val isFileSharingEnabled = isFileSharingEnabled().isFileSharingEnabled
+            if (isFileSharingEnabled != null) {
+                conversationViewState = conversationViewState.copy(isFileSharingEnabled = isFileSharingEnabled)
```

Comment:
Both the original and this code do exactly the same thing.

Unless the UseCase is mutating stuff (which it isn't).

Order of execution:
- `isFileSharingEnabled().isFileSharingEnabled`
- Check the if with the result of the last step
- `isFileSharingEnabled().isFileSharingEnabled` again
- call `copy` with the result of the last step
- update `conversationViewState` with the result from `copy`

There's no asynchronous behaviour at all in these calls.

There's no `launch`, `async`, no anything. This doesn't change anything.

Reply:
https://wearezeta.atlassian.net/browse/ACOL-101